backstage-helm/values.yaml

```diff
@@ -76,38 +77,14 @@ upstream:
                 frequency: { minutes: 1 }
                 timeout: { minutes: 1 }
                 initialDelay: { seconds: 15 }
-          github:
-            systems:
-              organization: 'janus-api-idp' # string
-              catalogPath: '/systems/*.yaml' # string
-              filters:
-                branch: 'main' # string
-                repository: 'demo-systems' # Regex
-              schedule: # optional
-                # supports cron, ISO duration, "human duration" as used in code
-                frequency: { minutes: 1 }
-                # supports ISO duration, "human duration" as used in code
-                timeout: { minutes: 1 }
-            components:
-              organization: 'janus-api-idp' # string
-              catalogPath: '/catalog-info.yaml' # string
-              filters:
-                branch: 'master' # string
-                repository: '.*' # Regex
-              schedule: # optional
-                # supports cron, ISO duration, "human duration" as used in code
-                frequency: { minutes: 1 }
-                # supports ISO duration, "human duration" as used in code
-                timeout: { minutes: 1 }
-            apis:
-              organization: 'janus-api-idp' # string
+          bitbucketServer:
+            yourProviderId: # identifies your ingested dataset
```

Comment:
the `yourProviderId` is no longer need based on my changes to the the main branch.  I'd add in what we did in the above github section that was removed

Reply:
done